.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto eol=lf
+
+*.bat text eol=crlf
+*.jar binary
```

Comment:
What do these options do?

Reply:
Mark `.bat` files ending with CRLF, it's the default EOL.